flit-client/src/exportedConstants/index.ts

```diff
@@ -0,0 +1 @@
+export { MSGSTATUS } from './status';
```

Comment:
I'm not sure we gain much breaking the different constant definitions out into their own files at the current size of the project. We don't incur a lot of cost in moving things out to files later, so keeping it all in the index file could keep the amount of boilerplate down.